apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx

```diff
@@ -0,0 +1,159 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+export const COLUMNS = {
+  NAME: 0,
+  NUM_MULTIPLE_CHOICE_CORRECT: 1,
+  NUM_MULTIPLE_CHOICE: 2,
+  PERCENT_CORRECT: 3,
+  SUBMISSION_TIMESTAMP: 4,
+};
+
+const studentOverviewDataPropType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  numMultipleChoiceCorrect: PropTypes.number.isRequired,
+  numMultipleChoice: PropTypes.number.isRequired,
+  percentCorrect: PropTypes.string.isRequired,
+  submissionTimestamp: PropTypes.string,
+  submissionStatus: PropTypes.string,
+});
+
+class StudentAssessmentOverviewTable extends Component {
+  static propTypes= {
+    studentOverviewData: PropTypes.arrayOf(studentOverviewDataPropType),
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        // Custom sortingOrder removes 'no-sort' from the cycle
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc'
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  submissionTimestampColumnFormatter = (submissionTimeStamp, {rowData}) => {
+    const submissionStatus = rowData.submissionStatus;
+
+    if (submissionStatus === 'Completed') {
+      return (
+        <div>
+          <div>
```

Comment:
i think you can remove the `div` around `submissionTimeStamp` and the checkmark since there aren't any extra styles provided by them

Reply:
^^^ sorry, my language is confusing: i think you can remove the individual `div`s around `submissionTimeStamp` and the checkmark (not the outer `div`)